docs/quick-ingestion-guides/powerbi/configuration.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Configuration
+---
+# Configuring Your PowerBI Connector to DataHub
+
+Now that you have created a DataHub specific Azure AD app with the relevant access in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a client id secret
+
+  This will securely store your PowerBI `Application (client) ID` within DataHub
+
+   * Enter a name like `POWER_BI_CLIENT_ID` - we will use this later to refer to the `Application (client) ID`
+   * Enter the `Application (client) ID`
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="Application (client) ID" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-client-id-secret.png"/>
+</p>
+
+4. Create a secret to store Azure AD app secret
+
+  This will securely store your  Azure AD app secret
+
+   * Enter a name like `POWER_BI_CLIENT_SECRET` - we will use this later to refer to the secret
+   * Enter the `Azure AD app` secret
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="Azure AD app Secret" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-client-secret.png"/>
+</p>
+
+## Configure Recipe
+
+1. Navigate to the **Sources** tab and click **Create new source**
+
+<p align="center">
+  <img width="75%" alt="Click &quot;Create new source&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_click_create_new_source_button.png"/>
+</p>
+
+2. Select PowerBI
+
+<p align="center">
+  <img width="70%" alt="Select PowerBI from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-source-window.png"/>
+</p>
+
+3. Fill out the PowerBI Recipe
+
+    Your PowerBI Tenant Id can be found at PowerBI Portal -> Click on `?` at top-right corner -> Click on `About PowerBI`  
```

Comment:
Is the `tenant id` the only required field?

Let's say something like this. 

"""
To complete the recipe, you'll need the following values:

- **tenant id**: <what is tenant id>
- **client id**: <what is client id>. You can use the secret you've previously created by providing the secret name surrounded by "${}". For example, "${CLIENT_ID_SECRET_NAME}"
- **client secret**


Considering 

"""

Reply:
done